Gemfile.lock

```diff
@@ -277,18 +323,18 @@ GEM
     websocket-driver (0.7.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
+    xpath (3.2.0)
+      nokogiri (~> 1.8)
     zeitwerk (2.5.4)
 
 PLATFORMS
+  x86_64-darwin-19
   aarch64-linux-musl
-  arm64-darwin-20
```

Comment:
Sorry, I meant the full group of platforms (not just the one I use)

Reply:
should all be back now